reconcile/utils/slack_api.py

```diff
@@ -285,22 +292,28 @@ def get_user_id_by_name(self, user_name: str) -> str:
 
         return result['user']['id']
 
-    def get_channels_by_names(self, channels_names):
-        return {k: v['name'] for k, v in self._get('channels').items()
+    def get_channels_by_names(self, channels_names: str):
+        return {k: v['name'] for k, v in self.get_channels()
                 if v['name'] in channels_names}
 
-    def get_channels_by_ids(self, channels_ids):
-        return {k: v['name'] for k, v in self._get('channels').items()
+    def get_channels_by_ids(self, channels_ids: str):
```

Comment:
Is this a string? I'd expect some `Iterable[str]` given the name of the channel and the arguments. Also, type hint for the return value.